resources/uplink_set.rb

```diff
@@ -0,0 +1,134 @@
+# (c) Copyright 2016 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+OneviewCookbook::ResourceBaseProperties.load(self)
+
+property :native_network, String       # Native network name - can be null
+property :networks, Array              # Array of strings containing Ethernet Network names - required
+property :fc_networks, Array           # Array of strings containing FC network names - required
+property :fcoe_networks, Array         # Array of strings containing FCoE Network names - required
+property :logical_interconnect, String # Associated logical interconnect name - required
```

Comment:
I figured out none of them are really necessary, as the default values are being set by the SDK. The only one that is not is nativeNetworkUri, which I manually set it as 'nil' in case it has not been added by the used. Please review it again! @jsmartt